tests/test_tftools.py

```diff
@@ -86,9 +86,9 @@ def test_batch_norm_2d():
     training = tf.placeholder(tf.bool, [])
     y = layers.batch_norm(x, training)
     sess = tf.Session()
-    sess.run(tf.initialize_all_variables())
-
     x_ = np.float32(np.random.randn(*input_shape))
+    
+    sess.run(tf.initialize_all_variables(), feed_dict={x:x_, training:True})
```

Comment:
I'd prefer marking that as a bugfix/todo to keep upgrades to new tf versions as simple as possible.